recipes/glue-astronomy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.1" %}
+
+package:
+  name: glue-astronomy
+  version: {{ version }}
+
+source:
+  fn: glue-astronomy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/glue-astronomy/glue-astronomy-{{ version }}.tar.gz
+  sha256: c31f8b1c95367a09c85e6719d4313fdb7a9acaf68d9abb96c6af68ebcdcde02d
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+
+  build:
+    - python
+    - pip
+    - setuptools >=30.3
+    - setuptools_scm
+
+  run:
```

Comment:
This package looks like it uses `pkg_resources` at runtime, so should it have `setuptools` under `run`?

Reply:
Done